src/main/java/games/strategy/engine/pbem/PBEMMessagePoster.java

```diff
@@ -28,37 +28,31 @@
 public class PBEMMessagePoster implements Serializable {
   public static final String FORUM_POSTER_PROP_NAME = "games.strategy.engine.pbem.IForumPoster";
   public static final String EMAIL_SENDER_PROP_NAME = "games.strategy.engine.pbem.IEmailSender";
-  public static final String WEB_POSTER_PROP_NAME = "games.strategy.engine.pbem.IWebPoster";
   public static final String PBEM_GAME_PROP_NAME = "games.strategy.engine.pbem.PBEMMessagePoster";
   private static final long serialVersionUID = 2256265436928530566L;
   private final IForumPoster m_forumPoster;
   private final IEmailSender m_emailSender;
-  private final IWebPoster m_webSitePoster;
```

Comment:
The removal of this field does not appear to cause any compatibility issues for the following reasons:

* Instances of `PBEMMessagePoster` **are not** part of a save game (verified by inspecting a PBEM/PBF save game).  Therefore, there is no possibility that the next release will attempt to read an incompatible serialized version of this class.
* Instances of `PBEMMessagePoster` **are** serialized when the `IAbstractForumPosterDelegate#postTurnSummary()` method runs (see the Javadoc for this type).  However, this appears to only occur on a single node within the context of a PBEM/PBF game.  Thus, there is no possibility of an incompatible instance having to be deserialized, as I don't believe this could could ever run, for example, during a network game.  Please advise if this is not the case.